### PR TITLE
Add timeline to roadmap

### PR DESCRIPTION
**Pull request by bmagyar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,27 @@
...
+| Adopt ROS2 managed node concept for lifecycle handling | ROS2 Foxy | M5 2020 | [ros2_control#43](https://github.com/ros-controls/ros2_control/issues/43) |
+| Integrate a new ros2_control verb into the ros2cli instead of the current CLI service interface. Examples: “ros2 control switch ...” , “ros2 control load ...” | ROS2 Foxy | M8 2020 | [ros2_control#29](https://github.com/ros-controls/ros2_control/issues/29) |
+| Achieve chaining controllers through ROS2 topics | ROS2 G | M8 2020 | [ros2_control#30](https://github.com/ros-controls/ros2_control/issues/30) |
+| Hard-real time guarantees | ROS2 G | M8 2020 |
+| Per-controller configurable update loop | ROS2 G | M8 2020 | [ros2_control#31](https://github.com/ros-controls/ros2_control/issues/31) |
+| Common emergency stop handler | ROS2 G | M8 2020 | |
+| Port all other controllers over. | ROS2 G | M8 2020 |
+| **Testing and validation on different robot platforms** | | | |
+| Real robot tests | ROS2 G | M8 2020 |
+| Tutorials with simulated robots? | ROS2 G | M8 2020 |
```
Comment (bmagyar) [inline README.md]: @jordan-palacios this section would be very thin without adding anything that gets delivered upstream so I've added tutorials with sim robots here, do you think that would fit the scope of validation?

**Comment by bmagyar** [comment]
@piraka9011 please check that the timeline works for you guys

File: README.md (hunk 1 above)
Comment (jordan-palacios) [reply README.md]: Yes, I agree.

**Comment by piraka9011** [comment]
LGTM, @bmagyar do you mind using a [table formatter](http://markdowntable.com/) and change links to [reference style](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links) to make it a bit more readable + easy to adjust in the future?

File: README.md (hunk 1 above)
Comment (bmagyar) [reply README.md]: Thanks!

**Comment by bmagyar** [comment]
Thanks for the review guys, I've created a follow-up issue to improve the appearance of the table. merging

**Pull request merged**